blackjack.py

```diff
@@ -79,23 +79,32 @@ def print_entire_hand(self):
         for card in self.hand:
             print card
 
+    def check_natural_win(self):
+        if len(self.hand) == 2:
+            if self.hand[0].value == 'Ace' and self.hand[1].value in ['Jack', 'Queen', 'King']:
```

Comment:
NIT: using `tuple`s instead of `list`s is a little bit faster and uses less memory:
```
            if self.hand[0].value == 'Ace' and self.hand[1].value in ('Jack', 'Queen', 'King'):
```

Reply:
I didn't know that, I'll change to tuples 🙂